pkgs/applications/science/logic/cadical/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "cadical";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "arminbiere";
+    repo = pname;
```

```suggestion
    repo = "cadical";
```

`repo` and `pname` are the same, but they don't necessarily change together.

---

Certainly. I did.